editor/js/Editor.js

```diff
@@ -671,11 +671,12 @@ Editor.prototype = {
 
 		this.signals.cameraResetted.dispatch();
 
-		this.history.fromJSON( json.history );
 		this.scripts = json.scripts;
 
 		this.setScene( await loader.parseAsync( json.scene ) );
 
+		this.history.fromJSON( json.history );
```

Comment:
The commands restore references from the scene in thei `fromJSON()` methods. This history must be restored after the scene graph has loaded otherwise all references into the scene graph are wrong.